core/src/components/tab-button/tab-button.ios.scss

```diff
@@ -136,4 +136,11 @@
 
 :host(.tab-layout-label-hide) ::slotted(ion-icon) {
   @include margin(0);
+}
+
+
+// iOS Tab Button: Translucent
+// --------------------------------------------------------------
+:host-context(ion-tab-bar.tab-bar-translucent) a:focus-visible {
```

Comment:
I don't think we can use this feature yet

Reply:
Which feature?